gcp-ts-gke-knative/README.md

```diff
@@ -0,0 +1,39 @@
+# gcp-ts-gke-knative
+All-in-one demo of Knative on GKE
+
+**Not intended for production use**
+
+This demo was adapted from the 
+[Knative installation guide](https://www.knative.dev/docs/install/knative-with-gke/#creating-a-kubernetes-cluster)
+for GKE and the related
+[example app](https://www.knative.dev/docs/install/getting-started-knative-app/)
+
+**Important Note**
+This demo creates a 3-node GKE cluster with `n1-standard-4` instances.
+Remember to tear down these resources after use to avoid unwanted charges.
+
+## Prerequisites
+
+* GCP account configured with the `Kubernetes Engine Cluster Admin` role
+
+## Known issues:
+
+1. Stack operations get very slow with so many resources in one stack.
+Splitting the deployment into  multiple stacks (e.g., GKE cluster,
+Istio/Knative, app) solves this problem, and is more representative
+of production use.
+
+1. `pulumi destroy` fails with an error:
```

Comment:
Can we fix this in the product or example before publishing this?  It seems a key goal we have here is to take this burden off of users.

Reply:
Yes, I haven't had time to dig into the root cause yet, but we should fix this in the k8s provider.